### PR TITLE
Allow yaxis yscale

### PR DESCRIPTION
**Pull request by ethanroday**
Addresses #240.

### CODE REVIEW

File: src/components/ChartRow.js [hunk 1]
```diff
@@ -171,6 +177,24 @@ export default class ChartRow extends React.Component {
...
+        // axis id.
+        this.scaleMap = {};
+        this.updateScales(this.props);
+    }
+
+    /**
+     * When we get changes to the row's props we update our map of
+     * axis scales.
+     */
+    componentWillReceiveProps(nextProps) {
```
Comment (sartaj10) [inline src/components/ChartRow.js]: Isn't there already a `componentWillReceiveProps` defined above?

File: src/components/ChartRow.js [hunk 2]
```diff
@@ -151,18 +144,31 @@ export default class ChartRow extends React.Component {
...
+                    // If necessary, initialize a ScaleInterpolator for this y-axis.
+                    // When the yScale changes, we will update this interpolator.
                     this.scaleMap[id] = new ScaleInterpolator(transition, easeSinOut, s => {
                         const yAxisScalerMap = this.state.yAxisScalerMap;
                         yAxisScalerMap[id] = s;
                         this.setState(yAxisScalerMap);
                     });
                 }
+                // Get the vertical scale for this y-axis.
+                let scale;
```
Comment (sartaj10) [inline src/components/ChartRow.js]: scale is defined above as well. Line 145

File: src/components/ChartRow.js (hunk 1 above)
Comment (sartaj10) [inline src/components/ChartRow.js]: Isn't there already a `componentWillReceiveProps` defined above? Line 129

File: src/components/ChartRow.js [hunk 3]
```diff
@@ -151,18 +144,31 @@ export default class ChartRow extends React.Component {
 
                 const scale = createScale(child, type, min, max, rangeBottom, rangeTop);
                 if (!_.has(this.scaleMap, id)) {
-                    // No scale map yet, create one on this.state.yAxisScalarMap
+                    // If necessary, initialize a ScaleInterpolator for this y-axis.
```
Comment (sartaj10) [inline src/components/ChartRow.js]: For line 139-142, you can replace the if condition with the `isChildYAxis(child)`

File: src/components/ChartRow.js (hunk 2 above)
Comment (ethanroday) [reply src/components/ChartRow.js]: It looks like something went wrong with the recent merge from master. If you look at the [previous state of the branch ](https://github.com/ethanroday/react-timeseries-charts/blob/61ec0ad268f1d020f121330699fd8d38538071b1/src/components/ChartRow.js), this issue doesn't appear.

File: src/components/ChartRow.js (hunk 1 above)
Comment (ethanroday) [reply src/components/ChartRow.js]: See above. There's only one `componentWillReceiveProps` in the previous state of the branch.

File: src/components/ChartRow.js (hunk 3 above)
Comment (ethanroday) [reply src/components/ChartRow.js]: Same here as above.

File: src/components/ChartRow.js (hunk 2 above)
Comment (sartaj10) [reply src/components/ChartRow.js]: Ah, yeah i believe that's the case :(

**Comment by sartaj10** [comment]
I fixed the issues while merging with master in #251

**Pull request closed without merging**